internal/dev_server/sdk/project_key_middleware.go

```diff
@@ -38,6 +39,7 @@ func GetProjectKeyFromAuthorizationHeader(handler http.Handler) http.Handler {
 	return http.HandlerFunc(func(writer http.ResponseWriter, request *http.Request) {
 		ctx := request.Context()
 		projectKey := request.Header.Get("Authorization")
+		projectKey = strings.TrimPrefix(projectKey, "api_key ") // some sdks set this as a prefix
```

Comment:
@keelerm84 Are there other scenarios we should be aware of for auth key formats?

Reply:
I'm not sure. The `api_key` prefix isn't coming from any of the SDKs I wouldn't think. We don't add additional prefixing, outside of the standard key formats.

Maybe that's something more aligned with the integrations team, or whoever is in charge of the auto-generated API SDKs?

Reply:
lol glad we were both surprised by this behavior. It looks like iOS _is_ doing this [prefixing thing](https://github.com/launchdarkly/ios-client-sdk/blob/cb270ec1080ba7001b1892137f72151cf7e77be1/LaunchDarkly/LaunchDarkly/Networking/HTTPHeaders.swift#L34). Looks like same for [android](https://github.com/launchdarkly/android-client-sdk/blob/2a902c8f656a5ab4bb692d0b595c8e85f9796097/launchdarkly-android-client-sdk/src/test/java/com/launchdarkly/sdk/android/LDConfigTest.java#L102).

Reply:
🤯 Well that IS surprising. I wonder if that is still necessary.

I checked the relay, and there is a `fetchAuthToken` method that only checks for that `api_key` prefix, so maybe we can be more confident that prefix is the only one we have to worry about.

Reply:
Ooh. Thank you for looking at the relay implementation. If that's all it supports, I think we're all good here.